Strategies/TransitionToPartner.md

```diff
@@ -28,26 +28,22 @@ It's a reference, with business terms, that development teams will use to quickl
 
 Any privacy concerns will be treated separately, outside of the Enterprise Information Model.
 
-The Enterprise Information Model is part of a broader undertaking by IITB's Information Architecture directorate that seeks to mature ESDC's overall Information Architecture capability.
+This aligns with key Government of Canada initiatives such as the Data Strategy, the Digital Exchange Platform, and the API Store.
 
-This also aligns with key Government of Canada initiatives such as the Data Strategy, the Digital Exchange Platform and the API Store.
+The Enterprise Information Model is part of a broader undertaking by either IITB's Information Architecture directorate, or the SSPB's Chief Data Office (CDO).
 
-## 2. Accelerate Cloud Adoption
+## 2. Move towards a Cloud Native organization
 
-IITB will aggressively seek opportunities to adopt cloud computing and reap the benefits of rapid elasticity and increased agility.
+ESDC will aggressively seek opportunities to adopt cloud computing and reap the benefits of rapid elasticity, increased agility, and self-serving abilities.
 
-Accelerating cloud adoption at ESDC will require the following:
+Moving ESDC to a Cloud Native organization will require the following:
 
-1. Any new software development must be developed as a cloud native application *AND* deployed to the public cloud.
+1. Any new software development must be developed as a cloud native software *AND* deployed to the public cloud.
 This includes up to Protected B data classification.
-2. Development teams (dev, ops, testing) must have access to a public cloud sandbox environment, with the following characteristics:
-    - dedicated and predictable funding,
-    - self-managed by the team with financial monitoring performed so that teams do not exceed their funding cap
-    - for experiments and continuous improvement purposes only. 
-      - No work in a sandbox environment is meant to reach production. 
-      - No sensitive data is to be stored or used in this sandbox.
-3. Ability to reach the cloud environment over the Internet, outside of the GCNet; enabling development to happen without government resources and freeing network bandwidth.
-4. Start using the cloud before all the governance pieces have been established.
+2. If building on premise, software must be built to natively support Cloud even if they are not ready to do so.
+3. Each Development teams (dev, ops, testing) must have a public cloud sandbox made available to them with no restrictions or connectivity to existing ESDC infrastructure, intended solely for learning, innovation, and experimentation.
```

Comment:
- Might want to specify the `no restrictions`: no restrictions on services or no $ caps?
- Maybe replace `or connectivity` by `and no connectivity` to avoid confusion?